Week3/homework/step2-4.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 function Dog() {
-  // add your code here
+  this.name = 'Fido';
+  this.color = 'white and brown';
+  this.numLegs = 4;
 }
 
 const hound = new Dog();
```

Comment:
Nice! Don't think we got to this in the curriculum yet. Should hit this in two weeks.